CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Change Log
 
+## 1.3.0
+
+- Enhanced [pyOCD](https://open-cmsis-pack.github.io/cmsis-toolbox/YML-Input-Format/#pyocd)
+and [J-Link GDB Server](https://open-cmsis-pack.github.io/cmsis-toolbox/YML-Input-Format/#j-link-server) debug configurations for multi-core
+debug, semihosting serial I/O via Telnet, and more.
+    - Requires [Arm CMSIS Solution extension v1.64.0](https://marketplace.visualstudio.com/items?itemName=Arm.cmsis-csolution) or later.
```

Comment:
I am not sure we need to duplicate this dependency as it is mentioned further down.